src/status_im/chat/subs.cljs

```diff
@@ -8,51 +8,91 @@
             [status-im.models.transactions :as transactions]
             [status-im.utils.platform :as platform]))
 
-(re-frame/reg-sub :get-chats :chats)
+(re-frame/reg-sub ::chats :chats)
+(re-frame/reg-sub ::access-scope->command-id :access-scope->command-id)
+(re-frame/reg-sub ::chat-ui-props :chat-ui-props)
```

Comment:
It's beyond the scope of this change, but we should get rid of these keyword-only subs too. 
They are bugs waiting to happen because if the key is missing from the database, sub parameters will be returned.

Reply:
will be taken care of in #6787

Reply:
actually what kind of bugs do you have in mind it would mean app-db is broken

Reply:
Yes, it would mean that something is broken. Either app-db or possibly the flow (e.g. sub intended to be consumed after log in actually consumed before log in).

In those cases sub should return an empty response so that we can clearly see the cause. Not return a vector of keywords and propagate the bug further down the code.